src/test/test-io-util.c

```diff
@@ -11,30 +11,31 @@
 static void test_sparse_write_one(int fd, const char *buffer, size_t n) {
         char check[n];
 
-        assert_se(lseek(fd, 0, SEEK_SET) == 0);
-        assert_se(ftruncate(fd, 0) >= 0);
-        assert_se(sparse_write(fd, buffer, n, 4) == (ssize_t) n);
+        ASSERT_OK_EQ_ERRNO(lseek(fd, 0, SEEK_SET), 0);
+        ASSERT_OK_ZERO_ERRNO(ftruncate(fd, 0));
+        ASSERT_EQ(sparse_write(fd, buffer, n, 4), (ssize_t) n);
```

Comment:
Please use `ASSERT_OK_EQ()` for `sparse_write()`, as it returns negative errno on failure, and should returns `n` on success.